pkg/util/pretty/ui.go

```diff
@@ -0,0 +1,19 @@
+package pretty
```

Comment:
pretty is not an intuitionistic package name, maybe you can call it 'ui' directly!

Reply:
Exactly, the modification has been completed.